NullPointerException when saving mzML

```
Caused by: java.lang.NullPointerException: Cannot invoke "java.io.File.getName()" because the return value of "org.eclipse.chemclipse.msd.model.core.IChromatogramMSD.getFile()" is null
	at org.eclipse.chemclipse.msd.converter.supplier.mzml.io.ChromatogramWriterVersion110.createMzML(ChromatogramWriterVersion110.java:84)
	at org.eclipse.chemclipse.msd.converter.supplier.mzml.io.ChromatogramWriterVersion110.writeChromatogram(ChromatogramWriterVersion110.java:74)
	at org.eclipse.chemclipse.msd.converter.supplier.mzml.converter.io.ChromatogramWriter.writeChromatogram(ChromatogramWriter.java:35)
	at org.eclipse.chemclipse.msd.converter.supplier.mzml.converter.ChromatogramExportConverter.convert(ChromatogramExportConverter.java:45)
	at org.eclipse.chemclipse.converter.chromatogram.AbstractChromatogramConverter.convert(AbstractChromatogramConverter.java:288)
```

--- a/chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzml/src/org/eclipse/chemclipse/msd/converter/supplier/mzml/io/ChromatogramWriterVersion110.java
+++ b/chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzml/src/org/eclipse/chemclipse/msd/converter/supplier/mzml/io/ChromatogramWriterVersion110.java
@@ -81,7 +81,6 @@ public class ChromatogramWriterVersion110 extends AbstractChromatogramWriter imp
 
 		MzMLType mzML = new MzMLType();
 		mzML.setVersion(XmlReader110.VERSION);
-		mzML.setId(chromatogram.getFile().getName());
 		SourceFileListType sourceFileList = createSourceFileList(chromatogram);
 		mzML.setFileDescription(createFileDescription(chromatogram, sourceFileList));
 		SoftwareListType softwareList = XmlWriter110.createSoftwareList();

--- a/chemclipse/plugins/org.eclipse.chemclipse.wsd.converter.supplier.mzml/src/org/eclipse/chemclipse/wsd/converter/supplier/mzml/io/ChromatogramWriterVersion110.java
+++ b/chemclipse/plugins/org.eclipse.chemclipse.wsd.converter.supplier.mzml/src/org/eclipse/chemclipse/wsd/converter/supplier/mzml/io/ChromatogramWriterVersion110.java
@@ -123,7 +123,6 @@ public class ChromatogramWriterVersion110 extends AbstractChromatogramWriter imp
 				run.setStartTimeStamp(date);
 			}
 			MzMLType mzML = new MzMLType();
-			mzML.setId(chromatogram.getFile().getName());
 			//
 			CVListType cvList = new CVListType();
 			cvList.setCount(BigInteger.valueOf(2));

